File: web/src/main/kotlin/com/isystk/sample/web/base/controller/html/BaseSearchForm.java
package com.isystk.sample.web.base.controller.html;

public abstract class BaseSearchForm extends BaseForm {

  private static final long serialVersionUID = -7129975017789825804L;

  int page = 1;

  int perpage = 10;

  public int getPage() {
    return this.page;
  }

  public int getPerpage() {
    return this.perpage;
  }

  public void setPage(int page) {
    this.page = page;
  }

  public void setPerpage(int perpage) {
    this.perpage = perpage;
  }
}

File: web/src/main/kotlin/com/isystk/sample/web/base/controller/html/BaseSearchForm.kt
package com.isystk.sample.web.base.controller.html

abstract class BaseSearchForm : BaseForm() {

    var page = 1
    var perpage = 10

    open fun page(): Int {
        return page;
    }

   open fun perpage(): Int {
        return perpage
    }

    companion object {
        private const val serialVersionUID = -7129975017789825804L
    }
}